### DISCUSSION TITLE
what is included in specific binaries?

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#20293] [Q&A]
Where can i find the 'uer_config_overwrite.h' for each different binaries?
i want to check what is included.

**Comment by barbudor** [comment]
Base tasmota is in 'my_user_config.h'
Then variants are in 'tasmota_configurations.h' for esp8266 and 'TASMOTA_CONFIGURATIONS_ESP32.h' for esp32

Order of inclusion is
my_user_config.h
user_config_override.h
tasmota_configurations(_ESP32).h

**Reply by barbudor** [reply]
And BUILDS.md is supposed to document what is in each variants

**Reply by sfromis** [reply]
Additionally, you can use the tool `decode-status` to get a listing of many of the available features included in the binary you're running. While this listing does not cover all details, it does give a general overview.

**Reply by ChrisVonFrosta** [reply]
where  can i find this tool? i tried this as command but ist unknown

**Reply by ChrisVonFrosta** [reply]
>And BUILDS.md is supposed to document what is in each variants

i'm missing `bluetooth` there. That i was espacially looking for

**Reply by ChrisVonFrosta** [reply]
I'm surprised that I can no longer configure an ultrasonic sensor (SR04). My Berry script polls it, and I don't remember changing the binary. But I don't want to rule that out. Installed is: 13.0.0(bluetooth).

**Reply by ChrisVonFrosta** [reply]
i would like to use bluetooth and SR-04 without compiling (to stay on update path). is there a predefined binary that offers that?

i can't find a sensor binary on [site](http://ota.tasmota.com/tasmota32/release/)

**Reply by sfromis** [reply]
The special Bluetooth binary is specifically cut down when it comes to lots of other features, meaning only very few sensors included.

As indicated on the BUILDS page, the "sensors" selection only applies to ESP8266, with ESP32, these are already included in the base build, but specialized builds are trimmed for their purpose.

Googling for "Tasmota" and "decode-status" I saw https://github.com/tasmota/decode-status as the first hit. And running that tool you can see a relatively short list of features.

Creating other feature mixes means doing custom builds.
